.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
```

Comment:
Note: I've modified this commit to no longer have the `repositories` section.

It is not needed for this gem.